integrationTests/vm/testInitializer.go

```diff
@@ -783,32 +786,35 @@ func CreateTxProcessorWithOneSCExecutorWithVMs(
 		return nil, nil, nil, nil, nil, err
 	}
 
-	intermediateTxHandler := &mock.IntermediateTransactionHandlerMock{}
-	argsNewSCProcessor := smartContract.ArgsNewSmartContractProcessor{
-		VmContainer:                         vmContainer,
-		ArgsParser:                          smartContract.NewArgumentParser(),
-		Hasher:                              testHasher,
-		Marshalizer:                         testMarshalizer,
-		AccountsDB:                          accnts,
-		BlockChainHook:                      blockChainHook,
-		PubkeyConv:                          pubkeyConv,
-		ShardCoordinator:                    shardCoordinator,
-		ScrForwarder:                        intermediateTxHandler,
-		BadTxForwarder:                      intermediateTxHandler,
-		TxFeeHandler:                        feeAccumulator,
-		EconomicsFee:                        economicsData,
-		TxTypeHandler:                       txTypeHandler,
-		GasHandler:                          gasComp,
-		GasSchedule:                         mock.NewGasScheduleNotifierMock(gasSchedule),
-		TxLogsProcessor:                     &mock.TxLogsProcessorStub{},
-		EpochNotifier:                       forking.NewGenericEpochNotifier(),
+	enableEpochs := config.EnableEpochs{
```

Comment:
this approach is dangerous as we won't have any feedback if the value wasn't set and this way we'll get default values (0). in the future, we might think to have a separate component that handles enable epochs and it's functions return `(uint32, error)`.